packages/next-templates/package.json

```diff
@@ -36,6 +36,7 @@
     "@utrecht/design-tokens": "1.0.0-alpha.607",
     "@utrecht/web-component-library-react": "1.0.2",
     "@utrecht/web-component-library-stencil": "1.1.0",
-    "clsx": "2.0.0"
+    "clsx": "2.0.0",
+    "prettier": "3.2.5"
```

Comment:
denk niet dat we preittier hoeven te installeren voor de next template package.json

Reply:
Volgens mij had ik dezelfde feedback bij verhuizing-doorgeven van Robbert. En dit had ik indd verwijderd. Waarschijnlijk staat het nog daar omdat dat de oude versie is en ik heb de nieuwe versie nog niet op verhuizing-form.

Reply:
Gedaan